templates/helpers/cleanJsString.js

```diff
@@ -0,0 +1,10 @@
+var jsesc = require('jsesc');
+
+// use jscs to make our strings javascript ready
+// remove script tags so it doesn't mess with our parent <script> tags
+module.exports = (value) => jsesc(
+  value,
+  {
+    wrap: true
+  }
+).replace(/<\/script/g, '<\\/script>');
```

Comment:
Let’s update jsesc to v1.1.0 and use `escapeEtago: true` instead of doing the `.replace` ourselves. 👍